pysindy/optimizers/stlsq.py

```diff
@@ -152,8 +148,8 @@ def _reduce(self, x, y):
             for i in range(n_targets):
                 if np.count_nonzero(ind[i]) == 0:
                     warnings.warn(
-                        """Sparsity parameter is too big ({}) and eliminated all
-                        coefficients""".format(
+                        "Sparsity parameter is too big ({}) and eliminated all "
+                        "coefficients".format(
                             self.threshold
                         )
```

Comment:
This should also be put on one line:
```suggestion
                        "coefficients".format(self.threshold)
```